test/CodeGenILTests/ReplicationTests.cs

```diff
@@ -0,0 +1,255 @@
+﻿using NUnit.Framework;
+using ProtoCore.Utils;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CodeGenILTests
+{
+    class ReplicationTests
```

Comment:
you should be able to inherit from microTest and you'll get the setup / teardown for free.

Reply:
done